compiler/rustc_mir/src/dataflow/move_paths/builder.rs

```diff
@@ -319,6 +319,8 @@ impl<'b, 'a, 'tcx> Gatherer<'b, 'a, 'tcx> {
             | StatementKind::AscribeUserType(..)
             | StatementKind::Coverage(..)
             | StatementKind::Nop => {}
+
+            StatementKind::CopyNonOverlapping(..) => todo!(),
```

Comment:
Since both arguments are pointers, and thus `Operand::Copy` essentially, there is nothing to do for this statement kind and you can move it into the nop-list